GSA/digitalgov.gov
# POAM updates for Jan'24

## Summary

<p>This PR handles security vulnerabilities for November 2023.</p>
<ul>
<li>Updated NPM dependencies</li>
</ul>
<p><strong>Before</strong>
7 Patch
4 Minor
6 Major</p>
<p><strong>After</strong>
0 Path and 0 Minor
6 Major - due to other dependencies not ready for update</p>
<h2>Dependencies</h2>

Dependency | Old | New
-- | -- | --
 @uswds/uswds  | 3.7.0  | 3.7.1
   autoprefixer  | 10.4.16  | 10.4.17
   eslint-plugin-import             | 2.29.0  |  2.29.1
   postcss                          | 8.4.31  | 8.4.33
   sass                             | 1.69.5  | 1.69.7
   sass-embedded                    | 1.69.5  | 1.69.7
   stylelint-order                   | 6.0.3  | 6.0.4
  eslint                           | 8.54.0  |   8.56.0
   eslint-config-prettier            | 9.0.0  | 9.1.0
   gulp-postcss                      | 9.0.1  | 9.1.0
   image-size                        | 1.0.2  | 1.1.1

### Preview

[Link to Preview]()



### Solution

### How To Test

1. Check that build runs with no errors and local site does not have any major visual regression

---

### Dev Checklist
